socorro/processor/rules/breakpad.py

```diff
@@ -63,188 +60,73 @@ def action(self, raw_crash, raw_dumps, processed_crash, processor_meta):
         processed_crash['minidump_sha256_hash'] = raw_crash['MinidumpSha256Hash']
 
 
-class ExternalProcessRule(Rule):
```

Comment:
I nixed this class and folded the bits into `BreakpadStackwalkerRule2015` and `JitCrashCategorizeRule` as follows:

1. moved `dot_save` to a module-level function
2. moved `_execute_external_process` to a module-level function that also takes as an argument `interpret_command_output`--both subclasses used this, but had different interpret functions
3. folded `_interpret_command_output` into `BreakpadStackwalkerRule2015` since that's the subclass that uses it; `JitCrashCategorizeRule` had its own
4. folded `save_results` into `JitCrashCategorizeRule` since that's the only one that used it
5. folded `action` into `JitCrashCategorizeRule`; `BreakpadStackwalkerRule2015` defined its own `action`